### ISSUE TITLE
Refresh of the vis after changing configuration parameters does not work for Ecore editor

### ISSUE THREAD

**Issue by anthonyanjorin** [eMoflon/emoflon-core#111]
It only works for the sample ecore editor.  Would be nice if it would work independently of the specific editor being used to edit the metamodel.

**Comment by anthonyanjorin** [comment]
Any progress on this issue?  Have you at least been able to reproduce?

**Comment by anthonyanjorin** [comment]
This issue is extremely irritating - have you been able to reproduce?

**Comment by mvdheram** [comment]
By changing configuration parameters do you mean changing the properties of the .ecore
 file created ??

**Comment by anthonyanjorin** [comment]
With configuration parameters I mean the drop-down menu you get when you click on the little upside down triangle in the top-right corner of the PlantUML view.  Parameters include options such as “show model details”, “show neighbourhood”, and so on.

There are actually two problems here:

(1) When the vis is opened for a new model (.xmi or .ecore), the currently set configuration parameters are simply ignored.  One has to change them to make them active :(
(2) If they are changed the triggered refresh only works for the .xmi vis not for the .ecore vis.

Can you reproduce this?

**Comment by mvdheram** [comment]
Yeah, so I tried to reproduce this by using FamiliesWithSiblings and FeatureModelConcise.  When I changed  configuration parameters like enabling "abbreviate labels" of .ecore for FamiliesWithSiblings  model, the changes are being reflected in "FeatureModelConcise" .ecore visualisation with visualisation showing abbreviated labels of the classes and the parameters remain unchanged.

**Comment by anthonyanjorin** [comment]
How about the very first time you open the editor?  The value of the configuration parameters are simply ignored.

Btw.  when you open an .ecore file (select it and choose open with ...), there are two possibilities to open it:

- with the sample ecore editor 
- with the ecore editor 

Sounds confusing I know, but these are two *different* editors.  The refresh works for the sample ecore editor but not for the ecore editor.

Please try to reproduce (1) and (2) in my previous comment.

**Comment by anthonyanjorin** [comment]
Ok just focus on (2)
